client/components/map.jsx

```diff
@@ -71,82 +56,75 @@ export default function Map(props) {
   const onMapLoad = React.useCallback(map => {
     mapRef.current = map;
   }, []);
-
   if (loadError) return 'Error loading maps';
   if (!isLoaded) return 'Loading Maps';
 
   const handleAddSoundscapeClick = () => {
     setModal(true);
   };
 
-  return <div>
-    <GoogleMap
-    mapContainerStyle={mapContainerStyle}
-    zoom={7}
-    center={center}
-    onClick={onMapClick}
-    onLoad={onMapLoad}
-    >
-
-      {markers.map(markers => (
-        <Marker
-          key={markers.soundscapeId}
-          position={{ lat: markers.lat, lng: markers.lng }}
-        onClick={() => {
-          setSelectedMarker(markers);
-        }}
-        />
-      ))}
-
-      {selectedMarker &&
-          <InfoWindow
-          position={{ lat: selectedMarker.lat, lng: selectedMarker.lng }}
-            onCloseClick={() => { setSelectedMarker(null); }}>
-            <div>
-              <h5>Title: {selectedMarker.title}</h5>
-              <p>Description: {selectedMarker.description}</p>
-              <div className={styles.player}>
-                <audio ref={audioPlayer} src={selectedMarker.fileUrl}></audio>
-                <button size='sm' className={styles.circle} onClick={togglePlayPause}>
-                  {isPlaying
-                    ? <PauseBtnFill className={styles.play}/>
-                    : <PlayBtnFill className={styles.pause} />
-                  }
-                </button>
+  return (
+    <div>
+      <GoogleMap
+      mapContainerStyle={mapContainerStyle}
+      zoom={7}
+      center={center}
+      onClick={onMapClick}
+      onLoad={onMapLoad}
+      >
+
+        {markers.map(markers => (
+          <Marker
+            key={markers.soundscapeId}
+            position={{ lat: markers.lat, lng: markers.lng }}
+            onClick={() => {
+              setSelectedMarker(markers);
+            }}
+          />
+        ))}
+
+        {selectedMarker &&
+            <InfoWindow
+            position={{ lat: selectedMarker.lat, lng: selectedMarker.lng }}
+              onCloseClick={() => { setSelectedMarker(null); }}>
+              <div>
+                <h5>Title: {selectedMarker.title}</h5>
+                <p>Description: {selectedMarker.description}</p>
+                  <AudioPlayer src={selectedMarker.fileUrl} />
               </div>
+            </InfoWindow> }
+
+        {marker.map(marker => (
+          <Marker
+            key={marker.locationId}
+            position={{ lat: marker.lat, lng: marker.lng }}
+            onClick={() => {
+              setSelected(marker);
+            }}
+            />
+        ))}
```

Comment:
Something weird is going on here with these `marker` variable names. I understand `selectedMarker`. That semantically makes sense to me. 

Try changing up your variable names for these different `marker` things to be more descriptive. I'm also wondering why we even need a `markers` and a `marker`. Is one of them a `newMarker` that hasn't been saved to the server yet? Try changing your variable names so that they prevent questions like this.